charts/pxc-db/production-values.yaml

```diff
@@ -158,7 +158,7 @@ pxc:
 haproxy:
   enabled: true
   size: 3
-  image: ""
+  image: "percona/percona-xtradb-cluster-operator:1.10.0-haproxy-8.0.25"
```

Comment:
is it feasible to rewrite image on every release?
Maybe it's worth to create a rendering helper function?

Reply:
There is a helper function and we were using it (images were left empty and they would autopopulate), but since these images have some "custom" name since we added them after the initial release so I decided to put these values and plan to remove them when we will have next release so they will be auto populated again.